libs/game/game.ts

```diff
@@ -17,6 +17,30 @@ namespace game {
         None
     }
 
+    /**
+     * The available colors for Arcade.
+     * To stay synchronized with https://github.com/microsoft/pxt/blob/stable8.5/webapp/src/components/ImageEditor/sprite/Palette.tsx#L98.
+     * NOTE: If the color pallette is changed, these values will change along with it.
+     */
```

Comment:
```suggestion
    // To stay synchronized with https://github.com/microsoft/pxt/blob/stable8.5/webapp/src/components/ImageEditor/sprite/Palette.tsx#L98.
    /**
     * The available colors for Arcade.
     * NOTE: If the color palette is changed, these values will change along with it.
     **/
```

nit: lets swap this so the dev pointer is a separate comment, as is it will show up in hovertext

<img width="442" alt="image" src="https://user-images.githubusercontent.com/5615930/226981781-91a4b182-8cdf-4196-8023-ecc48a6f69aa.png">
vs.
<img width="487" alt="image" src="https://user-images.githubusercontent.com/5615930/226983495-2716dbe1-c14b-4399-91f2-d91731ad1fb2.png">

also nit: pallette -> palette~